app/views/keywords/_keyword.html.slim

```diff
@@ -1,3 +1,9 @@
 tr
   td = keyword.keyword
   td = keyword.created_at.to_date
+  td.text-center
+    - if keyword.status == 'processing'
+      == fa_icon 'spinner spin', class: "text-secondary"
+    - elsif keyword.status == 'processed'
```

Comment:
do we have failed status 🤔

Reply:
Should have but I cannot think of scenarios to make `failed` status since job can retry and I'm using `ActiveRecord::Base.transaction`, if anything wrong with the database transaction then status will rollback to `in_queue`.

Do you have any suggestion?

Reply:
FYI the `retry` has the limitation, as I remember the default one is 25 times, so soon the job will stop and the state still as in progress is that normal 🤔

Reply:
Make sense, will make the change

Reply:
Fixed in e279f8ec9841d9200880eb905146a5ea76d205e0